pb-jelly-gen/codegen/codegen.py

```diff
@@ -1885,8 +1883,6 @@ def crate_from_proto_filename(
 
 LIB_RS_HEADER = """
 #![warn(rust_2018_idioms)]
-#![allow(clippy::float_cmp)]
```

Comment:
removing this `#![allow(...)]` didn't break anything, I didn't look that hard though for where this might be applicable